functions/collatzConjecture.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+let result = [];
+export default function collatzConjecture(number){
```

Comment:
Space before brace `{`. Makes the code more readable. Format the rest of your code this way.